scilpy/tracking/propagator.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, dataset, step_size, rk_order,  space, origin):
         """
         Parameters
         ----------
-        dataset: scilpy.image.datasets.DataVolume
+        dataset: scilpy.image.volume_space_management.DataVolume
```

Comment:
Replace dataset with volume ? And the objects Dataset to DataVolume ?

Reply:
Here and on lines 258 and 324.

Reply:
I'm not sure I understand what you mean. The class DataVolume is in scilpy.image.volume_space_management.DataVolume so this seems correct to me.

Reply:
I was talking about the parameters names ;) the full class name is good